app/styles/app.scss

```diff
@@ -725,74 +725,79 @@ hr {
 
 #preprint-form-subjects {
     .subject {
-        background-color: $brand-primary;
         display: inline-block;
         color: white;
         overflow: hidden;
         padding: 0 2rem;
-        margin-right: 2rem;
-        margin-bottom: 1rem;
-        border-radius: 2px;
-        transition: background-color 0.1s linear;
+        margin: {
+            right: 2rem;
+            bottom: 1rem;
+        }
         &:hover {
-            background-color: lighten($brand-primary, 10%);
-            & > b:after {
+            & > .subject-container, & > .fa {
```

Comment:
There do not appear to be any children of subject-container with class fa at present, unless I'm missing something... 

If that is still true after any CSS refactoring you do, we generally encourage removing any style rules that are completely unused.